triagebot.toml

```diff
@@ -227,7 +222,7 @@ trigger_files = [
 
 [autolabel."O-macos"]
 trigger_files = [
-    "library/std/src/os/macos"
+    "library/std/src/os/apple"
 ]
```

Comment:
Creating a more appropriate label for Apple-specific stuff is tracked in https://github.com/rust-lang/rust/pull/124146.